### PR TITLE
Replace imports of Cassandra and Redis Frames 

### LINKED ISSUE #454: replace all imports of `cassandra_protocol::frame::Frame` and `redis_protocol::resp2::prelude::Frame` with the CassandraFrame/RedisFrame reexport in src/protocols/mod.rs (not in the draft PR)

### PR DESCRIPTION
**Pull request by Claude-at-Instaclustr**
Fix to move to RedisFrame and CassandraFrame

closes #454

### CODE REVIEW

**Comment by conorbros** [comment]
Just being pedantic but I would:
- Change the title to `"replace all imports of cassandra_protocol::frame::Fra..."` so it makes sense in the Git history 
- Link the PR to the issue using the GitHub UI if you want to make it clear that this PR closes the issue

File: shotover-proxy/benches/chain_benches.rs [hunk 1]
```diff
@@ -50,7 +50,8 @@ fn criterion_benchmark(c: &mut Criterion) {
     }
 
     {
-        use redis_protocol::resp2::prelude::*;
+        use shotover_proxy::protocols::RedisFrame;
+        //use redis_protocol::resp2::prelude::*;
```
Comment (rukai) [inline shotover-proxy/benches/chain_benches.rs:54]: PR looking good!
Just need to get rid of these commented lines.

Comment (Claude-at-Instaclustr) [reply shotover-proxy/benches/chain_benches.rs]: removed

**Review by conorbros — APPROVED** [review]
LGTM! Apologies about that failing test, should be fixed when you rebase after #452 is merged

**Review by rukai — APPROVED** [review]
LGTM, thanks for doing this.
As conor mentioned, now that we've fixed the failures on main via https://github.com/shotover/shotover-proxy/pull/452 you just need to rebase on lastest main and then github will let us merge.

**Pull request merged**